### DISCUSSION TITLE
Welcome to flutter_ios_voip_kit Discussions!

### DISCUSSION THREAD

**Discussion by masashi-sutou** [masashi-sutou/flutter_ios_voip_kit#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gagandeep1408** [comment]
How I can change set text coming call type like incoming call is audio or video?
